### PR TITLE
Add summary

### PR DESCRIPTION
**Pull request by Yacubane**

### CODE REVIEW

File: src/views/main/components/Summary.js [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+      case 5:
+        return "Excellent air!";
+      case 4:
+        return "Good air quality!";
+      case 3:
+        return "Mediocre air quality";
+      case 2:
+        return "Poor air quality";
+      case 1:
+        return "Bad air quality";
```
Comment (piaskowyk) [inline src/views/main/components/Summary.js]: Are we able to change `switch` to `dict`? What are you think about this solution?

File: src/views/main/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect, useRef } from "react";
+import { connect } from "react-redux";
+import styles from "../../../style/main/components/Login.module.scss";
+import Input from "../../../components/Input";
+import Button from "../../../components/Button";
+import { FaRegTimesCircle } from "react-icons/fa";
+import { setLoginDialogVisibility } from "../../../redux/actions";
+
+function Login(props) {
+  console.log("ess " + props.visibility)
```
Comment (piaskowyk) [inline src/views/main/components/Login.js]: perhaps to remove on production

File: src/style/components/input.module.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      .mainInput {
+        width: 100%;
+        font-size: 15px;
+        border: none;
+        background: transparent;
+        border-bottom: 1px solid #aaa;
+      }
+      .mainInput:focus {
+        outline: none;
+        border-bottom: 1px solid #333;
```
Comment (piaskowyk) [inline src/style/components/input.module.scss]: Could we move static color to var.scss file ?

File: src/components/Menu.js [hunk 4]
```diff
@@ -27,14 +29,14 @@ class Menu extends React.Component {
...
+				<Button onClick={() => this.props.dispatch(setLoginDialogVisibility(true))}>
+					<div className="holder">
+						<div><FaUser className="menuIcon" /></div>
+						<div className="textAligner">Account</div>
+					</div>
+				</Button>
+				</div>
 				<Link className="link about" to="/test">
 					<Button>
 						<div className="holder">
```
Comment (piaskowyk) [inline src/components/Menu.js]: 4 spaces => 2 spaces

File: src/views/main/components/Login.js (hunk 2 above)
Comment (Yacubane) [reply src/views/main/components/Login.js]: please kill me

File: src/views/main/components/Summary.js (hunk 1 above)
Comment (Yacubane) [reply src/views/main/components/Summary.js]: yeah, it will take less space 👍

**Pull request merged**